### PR TITLE
Crud todolist

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
1) Added addTask, editTask and removeTask functions.
2) rearrange the index on delete.

Thanks

### CODE REVIEW

File: src/modules/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    this.todolist.splice((index - 1), 1);
+    const updatedTodoList = this.todolist.map((object) => {
+      if (object.index > index) {
+        const ind = object.index - 1;
+        return { ...object, index: ind };
+      }
+      return object;
+    });
+    this.todolist = updatedTodoList;
+  }
```
Comment (AnselemOdims) [inline src/modules/TodoList.js:23]: - [ ] Good job implementing the remove functionality but at the moment, it does not work quite and does not update the screen immediately after an item is deleted.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @sajibarafatsiddiqui , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/TodoList.js (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply src/modules/TodoList.js:23]: Hi The issue has been resolved.

Thanks

**Review by rotimiazeez — APPROVED** [review]
Hi @sajibarafatsiddiqui 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter error ✅️
- Interactive web app 💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**